nuxt.config.js

```diff
@@ -125,8 +125,33 @@ export default {
   },
 
   sitemap: {
-    hostname: `${SITE_URLPREFIX}${SITE_PREFIX}`,
-    routes: getLifeEvents().map((le) => `/${le}`),
+    hostname: "https://benefits-tool.usa.gov",
+    routes: [
```

Comment:
We can confirm with @erniedeeb, but I think the routes would look something like this

```suggestion
    routes: [ 
      "/death-of-a-loved-one",
      "/es/death-of-a-loved-one",
      "/disability",
      "/es/disability",
      "/retirement",
      "/es/retirement",
      "/404",
      "/es/404",
      "/",
      "/es",
    ],
```